### PR TITLE
Feat/customize

### PR DESCRIPTION
**Pull request by LeratoKgomo**

### CODE REVIEW

**Comment by LeratoKgomo** [comment]
@pieryvdl im not really sure why the other test is failing . I ran the flutter analyze on both branches locally and it doesn't give any problems

**Comment by MonikaTheiss** [comment]
I will send you a screenshot of your error

**Comment by LeratoKgomo** [comment]
@MonikaTheiss  the issues have been fixed may you please review

**Comment by MonikaTheiss** [comment]
Will review now 😊

File: lib/pages/customize_page.dart
Comment (MonikaTheiss) [inline lib/pages/customize_page.dart:1]: Thank you for making your code readable by adding comments 😃

**Comment by MonikaTheiss** [comment]
I have looked through the code. Just need to run it on my side quickly

**Review by pieryvdl — APPROVED** [review]
Code looks good.

**Review by MonikaTheiss — APPROVED** [review]

**Pull request merged**